openpype/hosts/blender/plugins/publish/integrate_blend.py

```diff
@@ -51,6 +51,7 @@ def process(self, instance):
         representations.update(instance.data.get("hero_representations", {}))
 
         # Run commands for all published representations
+        instance.data.setdefault("futures", [])
```

Comment:
I don't find `futures` to be a very explicit name, or I don't understand the logic behind it.

Reply:
Actually `future` is the official name of what is returned by `pool.submit`: https://docs.python.org/3/library/asyncio-future.html

Reply:
I was not familiar with it, if I understand correctly, it's the expected result of an async task, right? In that case the `future` mention makes sense, but how about `async_futures`, to explicit that this is related to asynchronous programming?

Reply:
future cannot be anything else than from async. Maybe `representations_futures`?

Reply:
Okay, fine by me ^^